transport_nantes/mobilito/views.py

```diff
@@ -122,17 +127,20 @@ def get_pages_to_visit(self) -> list:
         """
         # This returns a list of pages visited.
         pages_seen: list = self.get_seen_pages_from_cookie()
+        completed_tutorial_timestamp = None
         if self.request.user.is_authenticated:
             user = get_MobilitoUser(self)
             completed_tutorial_timestamp = user.completed_tutorial_timestamp
-        else:
+
+        if not completed_tutorial_timestamp:
             completed_tutorial_timestamp = datetime(
-                year=2022, month=8, day=3, tzinfo=timezone.utc)
+                year=2022, month=8, day=2, tzinfo=timezone.utc)
```

Comment:
Where do these magic numbers (some day in August 2022) come from?

Reply:
needed a date to start from and compare, not set in stone.